src/notification/extract.js

```diff
@@ -0,0 +1,223 @@
+/* @flow strict-local */
+import type { Notification } from './types';
+import type { JSONable, JSONableDict, JSONableInput } from '../utils/jsonable';
+import * as logging from '../utils/logging';
+
+/** JSONableInput dictionary. */
+// eslint-disable-next-line flowtype/type-id-match
+type JSONableInputDict = { +[string]: JSONableInput | void };
```

Comment:
You also add one of these in `jsonable`. That seems like a fine place -- import from there?

Reply:
... I swear I *did* that. (Fixed.)